hack/quick_install.sh

```diff
@@ -31,14 +31,15 @@ while getopts ":hsr" option; do
 done
 
 export ISTIO_VERSION=1.19.4
+export ISTIO_DIR=istio-${ISTIO_VERSION}
 export KNATIVE_SERVING_VERSION=knative-v1.10.1
 export KNATIVE_ISTIO_VERSION=knative-v1.10.0
 export KSERVE_VERSION=v0.12.0-rc1
 export CERT_MANAGER_VERSION=v1.3.0
 export SCRIPT_DIR="$( dirname -- "${BASH_SOURCE[0]}" )"
 
 cleanup(){
-  rm -rf istio-${ISTIO_VERSION}
+  rm -rf ${ISTIO_DIR}
   rm -rf deploy-config-patch.yaml
 }
 trap cleanup EXIT
```

Comment:
won't the cleanup function remove the directory during exit even if the script encounters errors ?

Reply:
Good catch. I moved it to after Istio has been installed successfully